sites/ogj/server/templates/index.marko

```diff
@@ -160,22 +160,23 @@ $ const adSlots = {
     <div class="col-lg-4 mb-block">
       <endeavor-content-query-section-list
         limit=4
-        section-alias="editors-perspective"
+        section-alias="covid-19"
         requires-image=false
         use-placeholder=false
-        header={ title: "Editor's Perspective", href: "/editors-perspective" }
+        header={ title: "COVID-19", href: "/covid-19" }
       />
     </div>
   </div>
 
   <div class="row">
     <div class="col-lg-4 mb-block">
-      <endeavor-published-content-query-list
-        query={
-          contentTypes: ["Video"],
-          limit: 4,
-        }
-        header={ title: "Videos", href: "/videos" }
+      <endeavor-content-query-section-list
+        limit=4
+        section-alias="pipelines-transportation/lng"
+        header={ title: "LNG", href: "/pipelines-transportation/lng" }
+        requires-image=false
+        use-placeholder=false
+        native-x={ placement: 'list1', aliases: ['pipelines-transportation/lng'], index: 3 }
```

Comment:
Use double quotes in marko templates please!

Reply:
coolio. I copied over the part from the pipelines section above and just added the "lng", so I'll update the rest of the template as well

Reply:
Ah, didn't see that! Yeah if you could that would be great -- one less thing to fix when we eventually port them over to the base-cms-websites/endeavor-business-media repo.

Reply:
just the section blocks, or should I update the ad pulls and imports too?

Reply:
Should be everything in a `.marko` file (excluding punctuation that actually needs single quotes). That style choice was made after we migrated away from this repo, so it's just to keep everything in line with the style guide for the other projects